packages/block-editor/src/components/block-alignment-toolbar/index.js

```diff
@@ -45,6 +45,10 @@ const DEFAULT_CONTROLS = [ 'left', 'center', 'right', 'wide', 'full' ];
 const DEFAULT_CONTROL = 'center';
 const WIDE_CONTROLS = [ 'wide', 'full' ];
 
+const POPOVER_PROPS = {
```

Comment:
What's the point of having this as a constant?

Reply:
It's the only way I found to get this prop to take.

Reply:
I think the advantage is that the object is not regenerated on each render. On expensive pure components, it could have an impact but maybe not everywhere.

Reply:
Happy to do which ever you agree on. Just haven't been able to get it to work without this method.

Reply:
It would be nice to unblock this PR, so we don't ship the current change in master if we can avoid it. CC: @youknowriad

Reply:
@jasmussen this shouldn't be a blocker IMO. You can inline these if needed. Performance is probably not a concern  here.